src/main/charts/confluence/templates/statefulset.yaml

```diff
@@ -71,6 +71,10 @@ spec:
             - name: ATL_TOMCAT_SECURE
               value: "true"
             {{ end }}
+            - name: UMASK
+              value: {{ .Values.confluence.umask | quote }}
+            - name: SET_PERMISSIONS
+              value: "false"
```

Comment:
Set this to false to prevent the Docker image from locking down the local-home permissions. Without this, fluentd can't read the files.